sharding-core/sharding-core-common/src/test/java/org/apache/shardingsphere/core/strategy/keygen/SnowflakeShardingKeyGeneratorTest.java

```diff
@@ -181,4 +223,16 @@ public void assertSetMaxTolerateTimeDifferenceMilliseconds() {
         props.setAccessible(true);
         assertThat(((Properties) props.get(keyGenerator)).get("max.tolerate.time.difference.milliseconds"), is((Object) "1"));
     }
+    
+    @Test
+    @SneakyThrows
+    public void assertSetMaxVibrationOffsetSuccess() {
```

Comment:
Do we need this test case? Does it duplicate with `assertGenerateKeyWithMaxVibrationOffsetBe1LessThanPowerOf2`?